reconcile/queries.py

```diff
@@ -911,7 +912,9 @@ def get_repos_gitlab_owner(server=''):
     server: url of the server to return. for example: https://github.com
     """
     code_components = get_code_components()
-    return [c['url'] for c in code_components
+
+    return [([c['url']], c['gitlabRepoOwners']['persistLgtm'])
```

Comment:
this shoudl be a dictionary, or a nametuple, or a custom class... for the sake of readability. I mean, `('htttps://foo/bar', True)` doesn't tell much outside of this method.